app/src/main/res/values-ja/strings.xml

```diff
@@ -64,6 +64,9 @@
 
     <!-- Contributors -->
     <string name="contributors">コントリビューター</string>
-    <string name="contributors_header">%d 人のコントリビューター</string>
     <string name="contributors_avatar">コントリビューターのプロフィール画像</string>
+    <plurals name="contributors_header">
+        <item quantity="one">%d 人のコントリビューター</item>
```

Comment:
nit: Japanese setting always uses only `other`. 🤷‍♂️ 
ref: https://qiita.com/mstssk/items/6697db6947918e58593b

Reply:
woops, thanks! good to know